helm/redis/values.yaml

```diff
@@ -5,6 +5,7 @@
 replicaCount: 1
 
 redisImage:
+  #repository: <your registry>
```

Comment:
Remove the comment. And for redis dsa image, which we build, i think we should same with others. Just add a new field:
```
image:
  repository: <your registry>
  pullPolicy: IfNotPresent
  # Overrides the image tag whose default is the chart appVersion.
  tag: ""
```

And remove the duplicated pull policy from redisImage.

Reply:
combine the redisimage and image field